cvat-canvas/src/typescript/canvas.ts

```diff
@@ -35,9 +35,9 @@ const CanvasVersion = pjson.version;
 interface Canvas {
     html(): HTMLDivElement;
     setZLayer(zLayer: number | null): void;
-    setup(frameData: any, objectStates: any[]): void;
+    setup(frameData: any, objectStates: any[], frameAngle: number): void;
```

Comment:
Function ``setup()`` is quite computationally hard. 
We should use ``rotate()`` instead.
We can rewrite ``rotate()`` in such way that it accepts one argument ``angle: number``, then adjusts it (something like that ``Math.round((angle % 360) / 90)  * 90`` and applies)

Enum ``Rotation`` in such case can be moved from ``cvat-canvas`` to ``cvat-ui``.